include/TrustWalletCore/TWAnySigner.h

```diff
@@ -5,10 +5,10 @@
 // file LICENSE at the root of the source code distribution tree.
 #pragma once
 
-#include "TWBase.h"
-#include "TWCoinType.h"
-#include "TWData.h"
-#include "TWString.h"
+#include <TrustWalletCore/TWBase.h>
+#include <TrustWalletCore/TWCoinType.h>
```

Comment:
Can we rename TrustWalletCore to WalletCore?

Reply:
just include folder?

Reply:
Include is too general, if other CPP project uses it, it is better to be the project name.  WalletCore is OK, but would this not be a breaking change?  Or not too many CPP projects using wallet core out there?

Reply:
Is include only headers files? Maybe rename to headers

Reply:
This is definitely for another PR, but something to improve in naming

Reply:
`TrustWalletCore` is also the iOS framework name, just Headers don't make any sense